Tests/Engine/BrokerageTransactionHandlerTests/BrokerageTransactionHandlerTests.cs

```diff
@@ -14,89 +14,118 @@
 */
 
 using System;
+using Moq;
```

Comment:
We haven't really thought out which library we would use for mocking, but I've heard/read good things about `Moq` and think its a great addition to the test project.